common.h

```diff
@@ -432,6 +435,9 @@ typedef int strlen_t;
 typedef size_t strlen_t;
 #define PHPREDIS_ZVAL_IS_STRICT_FALSE(z) (Z_TYPE_P(z) == IS_FALSE)
 #define PHPREDIS_GET_OBJECT(class_entry, z) (class_entry *)((char *)Z_OBJ_P(z) - XtOffsetOf(class_entry, std))
+
+#define REDIS_MAKE_STD_ZVAL(zv) do {} while(0)
```

Comment:
Why not `PHPREDIS_NOTUSED` instead of empty do/while loop?

Reply:
Sure, that's fine with me.

Reply:
Haha if only it were so easy.  I can't wait to remove php 5.x support though.  So much duplicated code.